.gitignore

```diff
@@ -9,3 +9,4 @@ doc/
 open-source/
 outputs
 tags
+CMakeLists.txt.user
```

Comment:
This is a minor change to exclude a common file that Qt Creator adds to cmake environments.